dotcom-rendering/src/web/components/PinnedPost.tsx

```diff
@@ -20,9 +20,14 @@ import { decidePalette } from '../lib/decidePalette';
 const pinnedPostContainer = (palette: Palette) => css`
 	border: 3px solid ${palette.border.pinnedPost};
 	padding-bottom: ${space[1]}px;
-	margin-bottom: ${space[9]}px;
 	position: relative;
 	background: ${neutral[100]};
+	${from.mobile} {
+		margin-bottom: ${space[6] + space[5] / 2}px;
```

Comment:
This feels a little bit unorthodox. `Space` represents the guardian space conventions so we should be able to use them but if not, its preferable to favour the raw px rather than create a calculation

Reply:
Thanks for the comment Anna, I changed this to raw pixels.